src/Microsoft.Azure.WebJobs.Host/Dispatch/IMessageHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Host.Listeners;
```

Comment:
nit: Can we remove these now?

Reply:
these xml comments require it
`<see cref="ListenerFactoryContext.GetDispatchQueue(IMessageHandler)"/>` --> how `IMessageHandler` is used
`<see cref="ITriggeredFunctionExecutor.TryExecuteAsync(TriggeredFunctionData, CancellationToken)"/>` --> its doppelganger